TESTS/Makefile.am

```diff
@@ -25,17 +25,17 @@ bug_142_gen_SOURCES = bug_142_gen.f
 
 if ICB
 icb_arpack_c_SOURCES = icb_arpack_c.c
-icb_arpack_c_LDADD = $(top_builddir)/SRC/libarpack$(LIBSUFFIX).la
+icb_arpack_c_LDADD = $(LDADD)
 icb_arpack_c_CPPFLAGS = $(AM_CPPFLAGS) -I$(top_builddir) -I$(top_srcdir)/ICB
 
 icb_arpack_cpp_SOURCES = icb_arpack_cpp.cpp
-icb_arpack_cpp_LDADD = $(top_builddir)/SRC/libarpack$(LIBSUFFIX).la
+icb_arpack_cpp_LDADD = $(LDADD)
```

Comment:
Try deleting these two LDADD lines instead.

Reply:
Just to clarify, $(LDADD) is the default, so you don't need to specify it on each target like this.  I don't know what problem this is trying to solve, so I can't comment on whether changing the value is the right thing to do.

Reply:
@turboencabulator : you are right, I've just reverted it (since trying to test pyarpack on ILP64, lots of problems show up and mix up at the same time... I though this could explain one of them : I understood later that I was wrong). For sake of symmetry, and if I get that well, I tryed to push 9ab2463